.github/workflows/maven.yml

```diff
@@ -0,0 +1,20 @@
+name: phoenix-connectors CI
+
+on: 
+  push:
+    branches: master
```

Comment:
We actually want this to be triggered on each PR, not after pushing the change, right?

Reply:
The next line pull-request will run it on every PR and push here will run it whenever we have pushed changes. Do we want to remove action from push ?

Reply:
Yeah I think `pull_request` is sufficient since there's not much of a point running tests after the changes have been pushed. But, let's document this in Phoenix-connectors saying that we need PRs instead of patches for this project to run the tests.

Reply:
Removed on push action as requested.